src/views/FormPage.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="form">
+    <form>
+      <adresse>
+        <fieldset>
+          <legend>Titre :</legend>
+          <input type="checkbox" id="mr" name="title" value="mr" checked />
+          <label for="coding">Mr</label>
+          <input type="checkbox" id="mrs" name="title" value="mrs" />
+          <label for="coding">Mrs</label>
+          <input type="checkbox" id="ms" name="title" value="ms" />
+          <label for="music">Ms</label>
```

Comment:
for="music" this is a copy pasta mistake I guess :) when you declare a `for` attribute you need to have a matching `id` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label